crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/initiate.rs

```diff
@@ -54,7 +52,7 @@ impl ShardedGossipLocal {
         {
             let id = rand::thread_rng().gen();
 
-            let gossip = ShardedGossipWire::initiate(intervals, id, agent_list);
```

Comment:
The doc for this field suggests it's supposed to be local agents in the list that gets sent here, but `agent_list` was all agents. So this is a functional change here